commands/backups/capture.js

```diff
@@ -17,6 +17,21 @@ function * run (context, heroku) {
       yield heroku.post(`/postgres/v0/databases/${db.id}/snapshots`, {host: host(db)})
     }))
   } else {
+    let dbInfo = yield heroku.request({
+      host: host(db),
+      method: 'get',
+      path: `/client/v11/databases/${db.id}`
+    }).catch(err => {
+      if (err.statusCode !== 404) throw err
+      cli.exit(1, `${cli.color.addon(db.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
+    })
+    console.log(dbInfo.info)
```

Comment:
Is this intentional? Looks like [my original comment](https://github.com/heroku/heroku-pg/pull/158#discussion_r142765161) no longer shows up, but this is just debugging info, right?

Reply:
no I thought I removed it

Reply:
ah looks like I never saved the file